ej_cli/sis/regex.py

```diff
@@ -0,0 +1,25 @@
+matches = {
```

Comment:
wouldn't it be better to compile it so as not to do it every time?
```python
re.compile(r'a regex')
````

Reply:
I will try it :100: